### PR TITLE
Implement high_vs_low collection, add configuration for it

### PR DESCRIPTION
**Pull request by Mothblocks**
I misunderstood how GBP collection worked after that Atomic PR.

This introduces a `collection_method` configuration method, with two parameters: `"sum"` (the old method) and `"high_vs_low"` (the new default, and was in the webhook). This one adds the highest positive value with the lowest positive value to produce your GBP.

### CODE REVIEW

File: src/points.test.ts [hunk 1]
```diff
@@ -49,3 +49,120 @@ describe("setBalance", () => {
...
+    test("sum", () => {
+        expect(
+            points.getPointsFromLabels(
+                {
+                    ...configuration,
+                    collection_method: "sum",
+                },
+                ["+5", "+3", "-1", "doesn't exist", "-5"],
+            ),
+        ).toBe(2)
```
Comment (ninjanomnom) [inline src/points.test.ts:154]: Shouldn't this be 0?

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

File: src/points.test.ts (hunk 1 above)
Comment (Mothblocks) [reply src/points.test.ts:154]: This is the `sum` collection method test.

Comment (ninjanomnom) [reply src/points.test.ts:154]: Ah woops

Comment (Mothblocks) [reply src/points.test.ts:154]: As in--`5 + 3 - 1 - 5 == 2`

**Pull request merged**